components/desktop/gnome2/gnome-mag/Makefile

```diff
@@ -51,3 +52,18 @@ build:		$(BUILD_32)
 install:	$(INSTALL_32)
 
 test:		$(NO_TESTS)
+
+REQUIRED_PACKAGES += developer/gcc/gcc-libgcc
```

Comment:
That's the GCC 4 Runtime package.  It is available in OI, at least in oi_151a8 and oi_151a9, but not in hipster.  Did somebody rename it incorrectly?  The situation now is that there is no name that will work correctly in REQUIRED_PACKAGES.  I can only delete that line.

Are their other similar packages?  I added them with `gmake REQUIRED_PACKAGES' as you suggested.  Is there another way?